ffv1.md

```diff
@@ -929,10 +933,10 @@ pseudo-code                                                   | type
 Line( p, y ) {                                                |
     if (colorspace_type == 0) {                               |
         for( x = 0; x < plane_pixel_width[ p ]; x++ )         |
-            Pixel( p, y, x )                                  |
+            Sample( p, y, x )                                 |
```

Comment:
IIRC part of @dwbuiten was pointing out the need to define a Pixel/Sample function. Not blocking this commit or PR though.